### DISCUSSION TITLE
got captcha suddenly

### DISCUSSION THREAD

**Discussion by red-apple38** [ultrafunkamsterdam/undetected-chromedriver#1813] [Q&A]
Code worked fine - last check was 03/03/2024

For this site:
https://www.immobilienscout24.de/

Now when I press search, I have to solve a captcha.

Requested URL:
https://www.immobilienscout24.de/Suche/de/hessen/giessen-kreis/giessen/wohnung-mieten

UC version 3.5.5
Chrome version 123.0.6312.106

**Comment by st681** [comment]
I did not click your links but use Nodriver with proxies and you will likely not have captcha. On some sites, you will not be able to get around captcha. There are HCaptcha, ReCaptcha, etc. providers everywhere who will solve your captchas for a fraction of a cent.
